lesson-21/index.js

```diff
@@ -0,0 +1,119 @@
+"use strict"
+
+
+//task1
+ 
+// const fibonacci = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987];
+// fibonacci.forEach(element => console.log(element));
+
+// function a(f) {
+//     console.log(f);
+//     return f
+// }
+
+// fibonacci.forEach(a);
+
+
+//task2
+
+// const users = ['Darya', 'Masha', 'Denis', 'Vitaliy', 'Polina', 'Anton'];
+// const newUsers = users.map((f) => f ='member' + ': ' + f);
```

Comment:
Тут не нужно так делать. map работает немного по-другому. Все что вернет функция-callback и будет новым элементом массива. Также забыл добавить нумерацию по заданию
```suggestion
// const newUsers = users.map((f, index) => `member ${index + 1}: ${f}`);
```